src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://java.sun.com/xml/ns/javaee"
+  xmlns:web="http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd"
+  xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd"
```

Comment:
Until I got to the XML config fun, I felt like this was pretty lightweight. I recognize `web.xml` is inevitable in a servlet environment. The hope is that we won't be configuring endless beans etc via XML config  :-)